README.md

```diff
@@ -100,31 +94,51 @@ should work on LX-branded zones as well.
     deployment (see
     [the CNS operator guide](https://github.com/joyent/triton-cns/blob/master/docs/operator-guide.md)).
     We'll assume for the sake of example here that the CNS suffix for the
-    DC is `us-west-1.triton.zone`.
+    DC is
```

Comment:
This seems like an unnecessary change. Here, the dc name is part of the sentence. It's not a code block example.